### PR TITLE
Changed 'Submit' text on buttons with something more meaningful

### PR DESCRIPTION
**Pull request by shubham76**
This is regarding #3735.
I have changed 'Submit' text on buttons with 'Save', 'Add' etc. Also made some other typo changes.
@stsewd & @davidfischer please have a look.

### CODE REVIEW

File: readthedocs/templates/projects/project_advertising.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h4>{% trans "Opting out" %}</h4>
       {% endblocktrans %}
 
       <form method="get" action="{% url "gold_subscription" %}">
-        <input type="submit" value="{% trans "I can contribute financially" %}" />
+        <input type="submit" value="{% trans "I can contribute financially." %}" />
```
Comment (stsewd) [inline readthedocs/templates/projects/project_advertising.html]: Not sure if it's ok to include periods within a button.

Comment (shubham76) [reply readthedocs/templates/projects/project_advertising.html]: It's more like an entire sentence. That's why included that. Should I change it back the way it was?

Comment (stsewd) [reply readthedocs/templates/projects/project_advertising.html]: I don't have the arguments or the knowledge for choosing one or the other, was just a question p: But in general the PR should only solve the original issue, if you consider something else is needed is better to open a new issue, so your PR isn't blocked by discussions like this :)

File: readthedocs/templates/projects/project_edit.html [hunk 2]
```diff
@@ -19,10 +19,10 @@
     </p>
     {{ form.as_p }}
     <p>
-      <input style="display: inline;" type="submit" value="{% trans "Submit" %}">
+      <input style="display: inline;" type="submit" value="{% trans "Save" %}">
       {% comment %}Translators: The 'or' here is in between 'Submit' and 'delete'.{% endcomment %}
       {% trans "or" %}
-       <a href="{% url "projects_delete" project.slug %}">{% trans "delete" %}</a>
+       <a href="{% url "projects_delete" project.slug %}">{% trans "delete project" %}</a>
```
Comment (RichardLitt) [inline readthedocs/templates/projects/project_edit.html]: Why isn't this capitalized, too?

File: readthedocs/templates/projects/project_edit.html [hunk 3]
```diff
@@ -19,10 +19,10 @@
     </p>
     {{ form.as_p }}
     <p>
-      <input style="display: inline;" type="submit" value="{% trans "Submit" %}">
+      <input style="display: inline;" type="submit" value="{% trans "Save" %}">
       {% comment %}Translators: The 'or' here is in between 'Submit' and 'delete'.{% endcomment %}
```
[the remark this answers is not in the document]
Comment (shubham76) [reply readthedocs/templates/projects/project_edit.html]: Oh! Yes. My bad.

File: readthedocs/templates/projects/project_advertising.html (hunk 1 above)
Comment (shubham76) [reply readthedocs/templates/projects/project_advertising.html]: Yes. You are right. I am kind of new here. Sorry! It won't happen again! :+1:

Comment (stsewd) [reply readthedocs/templates/projects/project_advertising.html]: Nothing to worried about. Thanks for your interest on contributing.

Comment (shubham76) [reply readthedocs/templates/projects/project_advertising.html]: It's a great learning for me. Thanks for your constant help :)

File: readthedocs/templates/core/project_details.html [hunk 4]
```diff
@@ -66,7 +66,7 @@ <h3>{% trans "Build a version" %}</h3>
           <option value="{{ version.slug }}">{{ version.slug }}</option>
         {% endfor %}
         </select>
-        <input type="submit" value="{% trans "Build" %}">
+        <input type="submit" value="{% trans "Build Project" %}">
```
Comment (agjohnson) [inline readthedocs/templates/core/project_details.html]: As this is a build version form, perhaps this should be `Build version` instead

File: readthedocs/templates/projects/integration_form.html [hunk 5]
```diff
@@ -18,6 +18,6 @@
       action="{% url 'projects_integrations_create' project_slug=project.slug %}">
     {% csrf_token %}
     {{ form.as_p }}
-    <input type="submit" value="{% trans "Add integration" %}">
+    <input type="submit" value="{% trans "Add Integration" %}">
```
Comment (agjohnson) [inline readthedocs/templates/projects/integration_form.html]: We don't use title casing on our buttons, or at least shouldn't

File: readthedocs/templates/projects/integration_list.html [hunk 6]
```diff
@@ -17,7 +17,7 @@
       <li>
         <a class="button"
            href="{% url 'projects_integrations_create' project_slug=project.slug %}">
-          {% trans "Add integration" %}
+          {% trans "Add Integration" %}
```
Comment (agjohnson) [inline readthedocs/templates/projects/integration_list.html]: No title case here too

File: readthedocs/templates/core/project_details.html (hunk 4 above)
Comment (shubham76) [reply readthedocs/templates/core/project_details.html]: Oh! Yes. You are right. I'll change it right away!

File: readthedocs/templates/projects/integration_form.html (hunk 5 above)
Comment (shubham76) [reply readthedocs/templates/projects/integration_form.html]: Oh! Okay. I'll revert those changes. :+1:

**Comment by agjohnson** [comment]
This looks good to me, thanks for your contribution! 🎉

**Pull request merged**